classes/i18n/LocaleConversion.php

```diff
@@ -64,30 +111,36 @@ public static function get3LetterIsoFromLocale(?string $locale): ?string
      */
     public static function getLocaleFrom3LetterIso(?string $iso3Letter): ?string
     {
-        assert(strlen($iso3Letter) === 3);
         $primaryLocale = Locale::getPrimaryLocale();
 
-        $candidates = [];
+        $alpha2Candidates = $localeCandidates = [];
+        $languages = self::getISO6392b();
+        foreach (reset($languages) as $languageRaw) {
+            if (($languageRaw['bibliographic'] ?? null) === $iso3Letter || $languageRaw['alpha_3'] === $iso3Letter) {
+                if (array_key_exists('alpha_2', $languageRaw)) {
```

Comment:
I'm reviewing from my mobile, so it's complex to analyze the json 😁
Doesn't it always provide the `alpha_2`? If not, ignore my comment.